### PR TITLE
Wishlist error message

### PR DESCRIPTION
**Pull request by RakeshJesadiya**
## Purpose of this pull request

This pull request (PR) indicated the Error Message for the invalid Request payload on Add Product, Update Product and Remove Product to the wishlist.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/graphql/mutations/add-products-to-wishlist.html
https://devdocs.magento.com/guides/v2.4/graphql/mutations/update-products-in-wishlist.html
https://devdocs.magento.com/guides/v2.4/graphql/mutations/remove-products-from-wishlist.html

### CODE REVIEW

File: src/guides/v2.4/graphql/mutations/add-products-to-wishlist.md [hunk 1]
```diff
@@ -224,3 +224,10 @@ Attribute |  Data Type | Description
 ### WishListUserInputError attributes {#WishListUserInputError}
 
 {% include graphql/wishlist-user-input-errors.md %}
+
+## Errors
+
+Error | Description
+--- | ---
+`The current user cannot perform operations on wishlist` | An unauthorized user (guest) tried to add a item to wishlist, or an authorized user (customer) tried to add a item to wishlist of another customer.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/mutations/add-products-to-wishlist.md:232]:
```suggestion
`The current user cannot perform operations on wishlist` | An unauthorized user (guest) tried to add an item to a wishlist, or an authorized user (customer) tried to add an item to a wishlist belonging to another customer.
```

File: src/guides/v2.4/graphql/mutations/add-products-to-wishlist.md [hunk 2]
```diff
@@ -224,3 +224,10 @@ Attribute |  Data Type | Description
...
+`The wishlist was not found.` | Provided field `wishlistId` is invalid or not exists for the customer.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/mutations/add-products-to-wishlist.md:233]:
```suggestion
`The wishlist was not found.` | Provided field `wishlistId` is invalid or does not exist for the customer.
```

File: src/guides/v2.4/graphql/mutations/remove-products-from-wishlist.md [hunk 3]
```diff
@@ -148,3 +148,10 @@ Attribute |  Data Type | Description
 ### WishListUserInputError attributes {#WishListUserInputError}
 
 {% include graphql/wishlist-user-input-errors.md %}
+
+## Errors
+
+Error | Description
+--- | ---
+`The current user cannot perform operations on wishlist` | An unauthorized user (guest) tried to add a item to wishlist, or an authorized user (customer) tried to add a item to wishlist of another customer.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/mutations/remove-products-from-wishlist.md:156]:
```suggestion
`The current user cannot perform operations on wishlist` | An unauthorized user (guest) tried to add an item to a wishlist, or an authorized user (customer) tried to add an item to a wishlist of another customer.
```

File: src/guides/v2.4/graphql/mutations/remove-products-from-wishlist.md [hunk 4]
```diff
@@ -148,3 +148,10 @@ Attribute |  Data Type | Description
...
+`The wishlist was not found.` | Provided field `wishlistId` is invalid or not exists for the customer.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/mutations/remove-products-from-wishlist.md:157]:
```suggestion
`The wishlist was not found.` | Provided field `wishlistId` is invalid or does not exist for the customer.
```

File: src/guides/v2.4/graphql/mutations/update-products-in-wishlist.md [hunk 5]
```diff
@@ -207,3 +207,10 @@ Attribute |  Data Type | Description
 ### WishListUserInputError attributes {#WishListUserInputError}
 
 {% include graphql/wishlist-user-input-errors.md %}
+
+## Errors
+
+Error | Description
+--- | ---
+`The current user cannot perform operations on wishlist` | An unauthorized user (guest) tried to add a item to wishlist, or an authorized user (customer) tried to add a item to wishlist of another customer.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/mutations/update-products-in-wishlist.md:215]:
```suggestion
`The current user cannot perform operations on wishlist` | An unauthorized user (guest) tried to add an item to a wishlist, or an authorized user (customer) tried to add an item to a wishlist of another customer.
```

File: src/guides/v2.4/graphql/mutations/update-products-in-wishlist.md [hunk 6]
```diff
@@ -207,3 +207,10 @@ Attribute |  Data Type | Description
...
+`The wishlist was not found.` | Provided field `wishlistId` is invalid or not exists for the customer.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/mutations/update-products-in-wishlist.md:216]:
```suggestion
`The wishlist was not found.` | Provided field `wishlistId` is invalid or does not exist for the customer.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
just a few grammatical issues

**Comment by RakeshJesadiya** [comment]
@BarnyShergold Fixed the suggested changes. Thank you.

**Review by BarnyShergold — APPROVED** [review]
Excellent - thank you

**Review by keharper — APPROVED** [review]

**Comment by keharper** [comment]
running tests

**Pull request merged**